tests/test_2837_ufunc_attrs_behavior.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+import awkward as ak
+
+
+def test():
+    x = ak.Array([1, 2, 3], behavior={"foo": "BAR"}, attrs={"hello": "world"})
+    y = ak.Array([8, 0, 9], behavior={"do": "re"}, attrs={"mi": "fa"})
+    z = x + y
+    assert z.attrs == {"hello": "world", "mi": "fa"}
+    assert z.behavior == {"foo": "BAR", "do": "re"}
+
+
+def test_unary():
+    x = ak.Array([1, 2, 3], behavior={"foo": "BAR"}, attrs={"hello": "world"})
+    y = -x
+    assert y.attrs is x.attrs
+    assert x.behavior is y.behavior
```

Comment:
```suggestion
    assert y.attrs == x.attrs
    assert x.behavior == y.behavior
```

Reply:
I think for now we _do_ want to do this, so that we catch places that we're unnecessarily copying metadata. If we change this in future it means fixing up these tests, but at least it won't silently regress.

Reply:
Okay, I get it, then.